packages/secret-numbers/karma.config.js

```diff
@@ -0,0 +1,39 @@
+const webpackConfig = require("./test/webpack.config");
+delete webpackConfig.entry;
+
+module.exports = function (config) {
```

Comment:
When I update the other packages to run unit tests in the browsers most of this will live in a shared config file.